src/consensus/aft/impl/execution_utilities.cpp

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Should we avoid `.cpp` files as much as possible?

Reply:
We are planning on going way from all headers at this point